torch/distributed/distributed_c10d.py

```diff
@@ -1328,6 +1328,11 @@ def broadcast_multigpu(tensor_list, src, group=None, async_op=False, src_tensor=
         None, if not async_op or if not part of the group
 
     """
+    warnings.warn(
+        "torch.distributed.broadcast_multigpu will be deprecated. If you must "
```

Comment:
link to docs? (https://pytorch.org/docs/master/distributed.html#collective-functions)

Reply:
Thanks. Added